src/main/java/com/cburch/logisim/circuit/CircuitHDLGeneratorFactory.java

```diff
@@ -631,8 +637,8 @@ public LineBuffer getModuleFunctionality(Netlist theNetList, AttributeSet attrs)
                * connection is to a bus or to a normal net
                */
               final var connectedNet = solderPoint.getParentNet().getBitWidth() == 1
-                    ? LineBuffer.format("{{1}}{{2}}", NET_NAME, theNets.getNetId(solderPoint.getParentNet()))
-                    : LineBuffer.format("{{1}}{{2}}{{<}}{{3}}{{>}}", BUS_NAME,
+                    ? LineBuffer.formatHdl("{{1}}{{2}}", NET_NAME, theNets.getNetId(solderPoint.getParentNet()))
```

Comment:
Maybe we shall make HDL pairs always available?

Reply:
I still like the two versions, even though it "introduced" some "bugs" :-)